espanol/hijo/hijo.md

```diff
@@ -2,7 +2,7 @@ Reflexionaba sobre si los consejos que había dado llevarían a su hijo a morir
 ¿Será que sus convicciones llevarían a una guerra que destruiria a su familia? 
 Quizás, pero de seguro no traería el mejor resultado.
 
-Casualmente apareció el hijo en aquel mismo momento.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.